Sources/PaymentsCore/Networking/Enums/Environment.swift

```diff
@@ -13,5 +13,13 @@ public enum Environment {
             return URL(string: "https://api.paypal.com")!
         }
     }
-    // swiftlint:enable force_unwrapping
+
+    public var graphqlURL: URL {
```

Comment:
```suggestion
    public var graphQLURL: URL {
```
It's doesn't look the best, but it is consistent with the casing we have for `graphQL`.